storage/feed/handler.go

```diff
@@ -31,6 +31,13 @@ import (
 	"github.com/ethersphere/swarm/storage/feed/lookup"
 )
 
+// GenericHandler is an interface which specifies funcs any feeds handler should use
+type GenericHandler interface {
+	Lookup(ctx context.Context, query *Query) (*cacheEntry, error)
```

Comment:
this is the signature as originally defined in the `feeds` package.

it seems strange to me that the `cacheEntry` type is not exported, since it is returned in the `Lookup` function, which is.

Reply:
good point